SocialCareCaseViewerApi/V1/Gateways/SearchGateway.cs

```diff
@@ -63,17 +63,14 @@ private List<SearchResult> FilterByNameAgainstAllRecords(PersonSearchRequest req
             var name = request.Name == null ? "" : $"{request.Name}";
```

Comment:
This will potentially concatenate all three values together, but I suppose client can manage that to ensure the query is sensible

Reply:
It will yes, this is part of the first bit, expansion. A follow up commit will remove the name from the request param.